src/commands/aqua.ts

```diff
@@ -14,23 +14,29 @@
  * limitations under the License.
  */
 
+import { readdir, readFile, writeFile } from "node:fs/promises";
```

Comment:
I suggest to have a single way to do imports from `fs`
Currently you have two different "ways" to do it:

```
import { readdir, readFile, writeFile } from "node:fs/promises";
```

and

```
import fsPromises from "node:fs/promises";
```

Reply:
yes I am migrating towards the first one gradually where I find it. The first one is better because it is more concise and imports automatically